### PR TITLE
Adding opentelemetry-propagator-b3 from pypi

### PR DESCRIPTION
**Pull request by twoodwark**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x]  Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by twoodwark** [comment]
@conda-forge-admin, please ping team

File: recipes/opentelemetry-propagator-b3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "opentelemetry-propagator-b3" %}
+{% set version = "0.17b0" %}
```
Comment (ocefpaf) [inline recipes/opentelemetry-propagator-b3/meta.yaml:2]: This is a beta release, right? Is there a stable version that we can add to the main channel? If not we need to follow: https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Comment (twoodwark) [reply recipes/opentelemetry-propagator-b3/meta.yaml:2]: Hi. There's no stable version, just like dependency https://anaconda.org/conda-forge/opentelemetry-api .

**Comment by twoodwark** [comment]
Hi @mariusvniekerk , would you be interested in being a maintainer here / on related opentelemetry feedstocks?

File: recipes/opentelemetry-propagator-b3/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/opentelemetry-propagator-b3/meta.yaml:2]: yeah none of the the opentelemetry things have stable releases yet.  It seems to the their zerover phase to append a b0 to everything

Comment (mariusvniekerk) [reply recipes/opentelemetry-propagator-b3/meta.yaml:2]: @ocefpaf This falls under the same policy as black so we luckily don't need to do something special

Comment (ocefpaf) [reply recipes/opentelemetry-propagator-b3/meta.yaml:2]: Yep. Thanks @mariusvniekerk! (Just wanted to check before other core member start screaming at me ;-p)

**Pull request merged**